### ISSUE TITLE
[K4] 4.0 Compatibility

### ISSUE THREAD

**Issue by afbora** [gearsdigital/enhanced-toolbar-link-dialog#96]
Plugin doesn't work correctly in Kirby 4.0.0-rc.4

**Comment by toddz262** [comment]
I haven't played with 4.0 yet but am reading up on it... the new [Writer field](https://getkirby.com/releases/4.0#writer-field-improvements) has a link feature: _"Link to internal pages & files with our new link field in the link dialog. No more guessing of URLs."_ Sounds like this may take the place of this plugin?

**Comment by gearsdigital** [comment]
Yes, this behavior is now included in the Kirby core. Which is good so 👍🏻

I don't think I'm going any further with this plugin. I'll keep supporting it for v3 but I'm not adding any new features.

Merry Christmas by the way 🎄
